bin/node/src/cli.rs

```diff
@@ -44,11 +44,17 @@ impl SubstrateCli for Cli {
     }
 
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
+        #[cfg(not(feature = "mainnet"))]
+        let default_chain = "testnet";
+        #[cfg(feature = "mainnet")]
+        let default_chain = "mainnet";
```

Comment:
```suggestion
        let default_chain = if cfg!(feature = "mainnet") { "mainnet" } else { "testnet" };
```

Reply:
Removed and using `testnet` as default value.